plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -548,7 +548,9 @@ class FossId internal constructor(
                 // When a scan is created with the optional property 'git_repo_url', the server automatically creates
                 // an 'ignore rule' to exclude the '.git' directory.
                 // Therefore, this rule will be created automatically and does not need to be carried from the old scan.
-                val filteredRules = rules.filterNot { it.type == RuleType.DIRECTORY && it.value == ".git" }
+                val filteredRules = rules.filterNot {
```

Comment:
Commit message:
- Typo in "somes".
- Maybe instead of "escape" say that some versions seem to use the literal pattern for a regular expression?
- Consider extracting the set to a variable outside of the loop implied by `filterNot`.